low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -16,22 +16,27 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com
 
 import errno
-import psutil
+import re
+import shlex
```

Comment:
> import shlex

I think shlex is not getting used anywhere.
can remove this import

Reply:
Done